extensions/amp-story/1.0/amp-story-quiz.js

```diff
@@ -28,8 +34,13 @@ const answerChoiceOptions = ['A', 'B', 'C', 'D'];
 /** @const {string} */
 const TAG = 'amp-story-quiz';
 
+// TODO(jackbsteinberg): Refactor quiz to extend a general interactive element class
+// and make this an enum on that class.
+/** @const {number} */
+const INTERACTIVE_ELEMENT_TYPE_QUIZ = 0;
```

Comment:
The `INTERACTIVE` prefix is very broad and could introduce confusion down the line. In the context of a story, it could be a tweet with the `interactive` attribute (can be expanded), or any link, button, call to action...

Can you use whatever you come up with for the trigger name as a prefix here too?

Reply:
Opted for `STORY_REACTION` instead